### DISCUSSION TITLE
新一轮的解锁在7月9日要开始了，怎样才能得到尽可能高的分进审核

### DISCUSSION THREAD

**Discussion by hankarry** [MlgmXyysd/Xiaomi-BootLoader-Questionnaire#198] [General]
最近买了个K70pro，出厂澎湃，有5级号，我知道小米目前正规的解锁方式只能答题，但是也不想去给那些损塞当韭菜。所以趁此想询问一下诸位，15分钟30题怎样才能尽可能的能拿上高分。靠gpt4o查或者下午考等题库还是有什么别的小方法，以及如何快速查题，直接复制会被清分警告。
另外，小米现在这个政策和题的难度确实肯定不仅仅是为了筛核心搞机用户了，请各位收一下自己的情绪。所以我们集思广益一下，总结一下经验。

祝各位都能在小米高考里取得好成绩。

**Comment by pxcat0721** [comment]
没办法，你米要查你的github成分，就算过85也很难过

**Comment by peasca** [comment]
无解，目前小米bl解锁考试完全是黑箱
1.答题结果不公布（不提供哪道题错误）
2.答题分数标准不公布（题目分值）
3.评分标准不公布
4.无法保证公正透明（体现在前期小米社区工作人员恶意修改问卷）
科举都比这玩意公正多了，最好不要用小米高考，考公这类的词，这是在侮辱高考和国考。
目前解锁bl估计只能上供丧葬费解决问题，或者当精神香港同胞，台湾同胞，澳门同胞，华人华侨（购买外版设备）或者换u，其他无解
